app/models/open_hour.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: open_hours
+#
+#  id         :bigint           not null, primary key
+#  close      :time
+#  day        :integer
+#  open       :time
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  seller_id  :bigint           not null
+#
+# Indexes
+#
+#  index_open_hours_on_seller_id  (seller_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (seller_id => sellers.id)
+#
+class OpenHour < ApplicationRecord
```

Comment:
Can you add an enum to map MON to 0, TUE to 1, etc. to match the Square API: https://developer.squareup.com/reference/square/objects/BusinessHoursPeriod#definition__property-day_of_week

enums: https://naturaily.com/blog/ruby-on-rails-enum
https://api.rubyonrails.org/v5.2.3/classes/ActiveRecord/Enum.html